doc/index.rst

```diff
@@ -9,100 +7,176 @@
+        :columns: 6
+        :class-header:
 
-        Tutorials
+        **How to use Matplotlib?**
         ^^^
+        .. toctree::
+            :maxdepth: 1
 
-        - :ref:`Quick-start guide <quick_start>`
-        - :doc:`Plot types <plot_types/index>`
-        - :ref:`users-guide-using`
-        - :doc:`External learning resources <users/resources/index>`
+
+            User guide <users/index.rst>
+            tutorials/index.rst
+            users/faq/index.rst
 
     .. grid-item-card::
         :padding: 2
+        :columns: 6
 
-        How-tos
+        **What can Matplotlib do?**
         ^^^
+        .. toctree::
+            :maxdepth: 1
+
+            plot_types/index.rst
+            gallery/index.rst
 
-        - :doc:`Example gallery <gallery/index>`
-        - :doc:`Matplotlib FAQ <users/faq/index>`
 
     .. grid-item-card::
         :padding: 2
+        :columns: 12
 
-        Understand how Matplotlib works
+        **Reference**
         ^^^
 
-        - :ref:`users-guide-explain` in the :doc:`Users guide
-          <users/index>` has a number of advanced topics.
+        .. grid:: 1 1 2 2
+            :class-row: sd-align-minor-center
 
-    .. grid-item-card::
-        :padding: 2
+            .. grid-item::
 
-        Reference
-        ^^^
+                .. toctree::
+                    :maxdepth: 1
 
-        - :doc:`API Reference <api/index>`
-        - :doc:`Axes API <api/axes_api>` for most plotting methods
-        - :doc:`Figure API <api/figure_api>` for figure-level methods
-        - Top-level interfaces to create:
+                    API reference <api/index>
+                    Figure methods <api/figure_api>
+                    Plotting methods <api/axes_api>
+
+
+            .. grid-item::
+
+                Top-level interfaces to make:
+
+                - figures: `.pyplot.figure`
+                - subplots: `.pyplot.subplots`, `.pyplot.subplot_mosaic`
+
+
+Ecosystem
+=========
+
+.. grid:: 1 1 2 2
+    :class-row: sd-align-minor-center
+
+    .. grid-item::
+
+        .. toctree::
+            :maxdepth: 2
+
+            users/resources/index.rst
+
+    .. grid-item::
 
-          - Figures (`.pyplot.figure`)
-          - Subplots (`.pyplot.subplots`, `.pyplot.subplot_mosaic`)
+        :octicon:`link-external;1em;sd-text-info` `Third-party packages <https://matplotlib.org/mpl-third-party/>`_,
 
+        provide custom, domain specific, and experimental features, including
+        styles, colors, more plot types and backends, and alternative
+        interfaces.
 
-********************
-Third-party packages
-********************
 
-There are many `Third-party packages
-<https://matplotlib.org/mpl-third-party/>`_ built on top of and extending
-Matplotlib.
 
 
-************
-Contributing
-************
+About
+=====
+
+.. grid:: 1 1 2 2
+    :class-row: sd-align-minor-center
+
+    .. grid-item::
+
+        .. raw:: html
+
+            <div class="grid__intro" id="image_rotator"></div>
```

Comment:
This feels out of place. What does this have to do with "About"?

If you want to have something visual, I suggest to either use the matplotlib logo, or if that's too colorful here, some sort of "about icon".

Reply:
> was thinking more plots are what the library is about,

Yes, but plots is not what this section is about. This section is a kind of meta-section about the library itself. There are dedicatedly all non-plotting aspects, which is why I find plots inappropriate here.

More generally/aesthetically: it's a bit imbalanced that this is the only non-text entry.

Reply:
I'm having a lot of trouble with the idea that the only plots on this page would be the old-timey plots above.  We got rid of those because they were visually unappealing, and I don't think anyone visiting the page is going to know enough history to understand we are making a historical reference.  

I think having a plot on this page is in general a good idea.  I'd actually put it at the top, next to Install, which doesn't need to be so wide.  

The "About" section could have a picture if you really want.  Perhaps made to look like a screen grab from an old browser or something.   But I'm not sure we need "About" on this page at all?  We do have a brochure site...

Reply:
> Don't want it next to Install 'cause it'll mess with finding it on the page. One of the things I've grown to really embrace is the 'on this page' menu - unfortunately I haven't yet figured out how to make headings in grids and cards show up there.

Just put the heading `Installation` outside the columns/grids?

Reply:
If you want it showing up in "On this page" then I guess it needs to be a heading?  I don't think we need it on this page, but...

Reply:
Your eeg example isn't in the right spot or didn't get committed to static.  

However I still really think it's a bad idea to put this plot on the top level of the docs as the only example of what we can do. I understand the logic, but it is a poor visualization, both practically and aesthetically.

Reply:
The advantage of the plot type images is they are all the same size and the same style. I think pulling random Gallery images would be hard, though if you had a curated list that may work. 

The plot type rotator is on the front page,  but I think would be useful here as well ( drives traffic to Plot Types and shows basic functionality). I would still have it near the top.